server_admin/topics/sessions/transient.adoc

```diff
@@ -0,0 +1,11 @@
+[[_transient-session]]
+
+=== Transient sessions
+
+{project_name} has concept of transient sessions. When transient sessions are used, there is no real user session created after successful authentication.
+There is just temporary "transient" session created for the scope of the current request, which successfully authenticated user. This allows
+to run <<_protocol-mappers, protocol mappers>> after the authentication.
+
+When transient sessions are used, the client application does not have a way to refresh token, introspect token or check anyhow if particular session is valid.
+However this is not needed in some cases and hence the additional overhead needed for persistence of user sessions can be avoided.
```

Comment:
Suggest:
In some situations, these actions are not needed, so you can avoid the additional overhead for persistence of user sessions.